jwst/fits_generator/objects.py

```diff
@@ -568,6 +565,6 @@ def describe(self, stream, state):
             assert isinstance(hdudef, HDU)
             state.hdu = i
             state.depth += 1
-            hdu = hdudef.describe(stream, state)
+            hdu = hdudef.describe(stream, state) # noqa F841
```

Comment:
should this variable have been used somewhere?

Reply:
It looks to me like the function call is needed, but the return value is not.  It is probably equivalent to change the line to just `hdudef.describe(stream, state)`.

Reply:
```suggestion
            hdudef.describe(stream, state) # noqa F841
```